src/main/java/com/thealgorithms/sorts/SortUtils.java

```diff
@@ -116,4 +116,26 @@ public static <T extends Comparable<T>> boolean isSorted(List<T> list) {
         }
         return true;
     }
+
+    /**
+     * Finds the maximum element in the given array.
+     *
+     * @param <T> the type of elements in the array, which must implement the Comparable interface
+     * @param array the array to be searched
+     * @return the maximum element in the array
+     * @throws IllegalArgumentException if the array is null or empty
+     */
+    public static <T extends Comparable<T>> T max(T[] array) {
+        if (array == null || array.length == 0) {
+            throw new IllegalArgumentException("Array must not be null or empty");
+        }
```

Comment:
When this method is marked as `private` I am not really sure if this is needed. I do not have a strong opinion.